File: resolucoes/JavaScript/lista01/ex012.js
const prompt = require('prompt-sync')();

function main() {

const preco = Number(prompt("Digite o valor do produto:"))

const desconto = preco * 0.05

const promocao = preco - desconto

console.log(`O valor do seu podre é: ${preco} e seu desconto de 5% é ${promocao}`)
}

module.exports = {
    main
}

File: resolucoes/kotlin/lista01/src/main/kotlin/ex012.kt
import java.util.*

internal fun calcularPrecoPromocional() {
    val scanner = Scanner(System.`in`)
    print("Digite o preço do produto: ")
    val preco = scanner.nextDouble()

    // Calcula o valor do desconto
    val desconto = preco * 0.05
    // Calcula o preço promocional
    val precoPromocional = preco - desconto

    // Exibe o preço promocional
    println("O preço promocional é: R$${precoPromocional}")
}

fun main() {
    calcularPrecoPromocional()
}
